tests/sync/app.cpp

```diff
@@ -1783,8 +1789,11 @@ TEST_CASE("app: sync integration", "[sync][app]") {
 
     // MARK: Expired Session Refresh -
     SECTION("Expired Session Refresh") {
+        TestSyncManager& sync_manager = *new TestSyncManager({
```

Comment:
If this is needed it definitely needs a comment explaining why it's doing this.

Reply:
It is not needed, I'll change it.